packages/mds-repository/naming-strategies/mds-naming-strategy.ts

```diff
@@ -26,4 +26,8 @@ export class MdsNamingStrategy extends DefaultNamingStrategy implements NamingSt
   indexName(tableOrName: Table | string, columnNames: string[], where?: string): string {
     return ['idx', ...columnNames, tableName(tableOrName)].join('_')
   }
+
+  uniqueConstraintName(tableOrName: Table | string, columnNames: string[]): string {
```

Comment:
TypeORM supports naming strategies that allow you to control the names it generates. So, you can get something predictable/readable by overriding the naming strategy. In this case you would get something like `uc_name_posts` rather than `UC_{some generated UUID}`. Mostly our naming strategies mimic what the `mds-db` schema uses to make it easier to drop in compatible TypeORM models at some point down the road if we need to.